tests/ops/test_qubit_ops.py

```diff
@@ -2840,10 +2840,10 @@ def test_qubit_unitary_decomposition(self, U, expected_gate, expected_params):
         assert np.allclose(decomp[0].parameters, expected_params)
 
     def test_qubit_unitary_decomposition_multiqubit_invalid(self):
-        """Test that QubitUnitary is not decomposed for more than a single qubit."""
-        U = qml.CRZ(0.3, wires=[0, 1]).matrix
+        """Test that QubitUnitary is not decomposed for more than two qubits."""
+        U = qml.Toffoli(wires=[0, 1, 2]).matrix
```

Comment:
😆

Reply:
(_are_ there any known, systematic, 3-qubit decompositions?)

Reply:
Yes! Actually a new one just came out yesterday: https://arxiv.org/abs/2109.06770v1

But after having finally finished this transform, I don't even want to think about programming a harder one :laughing:

Reply:
I was going to make a joke about making a 'good first issue' for implementing that paper, but then felt bad 😅

Reply:
Too soon :weary: